WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -72,7 +72,7 @@ class AuthenticationManager: Authentication {
                                                 navBarBadgeColor: .primary,
                                                 navBarBackgroundColor: .appBar,
                                                 prologueTopContainerChildViewController: LoginPrologueViewController(),
-                                                statusBarStyle: .default)
+                                                statusBarStyle: .lightContent)
```

Comment:
I think this line of change might have caused the status bar on the prologue screen to be invisible in Light mode:

(the time and network/battery status are not visible)

PR branch | `develop` or if I changed this line back to `.default`
-- | --
![Simulator Screen Shot - iPhone 11 - 2020-11-05 at 09 59 49](https://user-images.githubusercontent.com/1945542/98188572-33afc700-1f4e-11eb-8025-c14322e54c3e.png) | ![Simulator Screen Shot - iPhone 11 - 2020-11-05 at 10 00 20](https://user-images.githubusercontent.com/1945542/98188580-36aab780-1f4e-11eb-9cb9-75cffd13c005.png)